data/content-versions.json

```diff
@@ -2,116 +2,68 @@
-      "214": "bf6ec4bd",
-      "215": "bf6ec4bd",
-      "230": "bf6ec4bd",
-      "232": "bf6ec4bd",
-      "235": "bf6ec4bd",
-      "240": "993d53d9",
-      "241": "993d53d9",
-      "242": "993d53d9",
-      "243": "993d53d9",
-      "244": "993d53d9",
-      "245": "993d53d9",
-      "246": "993d53d9",
-      "247": "993d53d9",
-      "248": "993d53d9",
-      "249": "993d53d9",
-      "250": "993d53d9",
-      "251": "993d53d9",
-      "252": "993d53d9",
-      "253": "993d53d9",
-      "254": "993d53d9"
+      "256": "animate-lomp-variant-a",
+      "257": "animate-lomp-variant-a",
+      "258": "animate-lomp-variant-a",
+      "259": "animate-lomp-variant-b",
+      "260": "animate-lomp-variant-a",
+      "261": "animate-lomp-variant-a",
+      "262": "animate-lomp-variant-a",
+      "263": "animate-lomp-variant-a",
+      "264": "animate-lomp-variant-b",
+      "265": "animate-lomp-variant-b",
+      "266": "animate-lomp-variant-b",
+      "267": "animate-lomp-variant-a",
+      "268": "animate-lomp-variant-b",
+      "269": "animate-lomp-variant-b",
+      "270": "animate-lomp-variant-a",
+      "271": "animate-lomp-variant-a",
+      "272": "animate-lomp-variant-b",
+      "273": "animate-lomp-variant-a",
+      "274": "animate-lomp-variant-b",
+      "275": "animate-lomp-variant-a",
+      "276": "animate-lomp-variant-a",
+      "277": "animate-lomp-variant-a",
+      "278": "animate-lomp-variant-b",
+      "279": "animate-lomp-variant-a",
+      "280": "animate-lomp-variant-a",
+      "281": "animate-lomp-variant-b",
+      "282": "animate-lomp-variant-a",
+      "283": "animate-lomp-variant-b",
+      "284": "animate-lomp-variant-b",
+      "285": "animate-lomp-variant-b",
+      "286": "animate-lomp-variant-b",
+      "287": "animate-lomp-variant-b",
+      "288": "animate-lomp-variant-a",
+      "289": "animate-lomp-variant-b",
+      "290": "animate-lomp-variant-b",
+      "291": "animate-lomp-variant-b",
+      "292": "animate-lomp-variant-b",
+      "293": "animate-lomp-variant-a",
+      "294": "animate-lomp-variant-a",
+      "295": "animate-lomp-variant-b",
+      "296": "animate-lomp-variant-b",
+      "297": "animate-lomp-variant-b",
+      "298": "animate-lomp-variant-a",
+      "299": "animate-lomp-variant-b",
+      "300": "animate-lomp-variant-a",
+      "301": "animate-lomp-variant-a",
+      "302": "animate-lomp-variant-b",
+      "303": "animate-lomp-variant-b",
+      "304": "animate-lomp-variant-b",
+      "305": "animate-lomp-variant-a",
+      "306": "animate-lomp-variant-a",
+      "307": "animate-lomp-variant-b",
+      "308": "animate-lomp-variant-b",
+      "309": "animate-lomp-variant-a",
+      "310": "animate-lomp-variant-a",
+      "311": "animate-lomp-variant-a",
+      "312": "animate-lomp-variant-a",
+      "313": "animate-lomp-variant-a",
+      "314": "animate-lomp-variant-b",
+      "315": "animate-lomp-variant-a",
+      "316": "animate-lomp-variant-a",
+      "317": "animate-lomp-variant-b"
```

Comment:
Perhaps this is intentional, but this isn't currently mapping all of the existing AY23 courses. Also, perhaps we shouldn't map id `256` since that is a canary course.